recipes/pyqtads/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyqtads" %}
+{% set version = "2.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/archive/{{ version }}.tar.gz
+  sha256: b0c83c4ced7ad25d0a090e9fa854ff7cc957523004c72046b60ab8a9d12874f5
+  patches:
+    - 0001-setup.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - pyqt >=5.9
+    - qt
+    - setuptools
+    - sip >=4.19
+  run:
+    - python
+    - pyqt >=5.9
+    - pywin32   # [win]
+    - qt
+    - sip >=4.19
+
+test:
+  imports:
+    - PyQtAds
+
+about:
+  home: https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System
+  license: LGPLv2+ 
+  license_family: LGPL 
+  license_file: LICENSE.md
+  summary: 'Advanced Docking System for Qt'
+
+  description: 'Advanced Docking System for Qt' 
+  dev_url: https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System
+
+extra:
+  recipe-maintainers:
+    - hhslepicka
+    - klauer
```

Comment:
@klauer , do you agree to maintain this recipe as well?

Reply:
Yes, I will help with it.